RELEASE_NOTES.md

```diff
@@ -14,6 +14,7 @@
 - Address Lookup not displaying validation error on Card Component when no address has been selected.
 - Actions no longer crash when your app uses obfuscation.
 - Drop-in no longer throws an error while handling a 3DS2 challenge on API 66 and below.
+- When the app process is killed after calling `handleAction`, then the state will now be restored and `handleIntent` can be called to continue the payment process.
```

Comment:
I think we can make this a bit more broad, since we fixed many issues related to activity state restoration

Reply:
Something like this?
```
When the app process is dies during action handling, then the state will now be restored and the payment process can be continued.
```

Reply:
👍